lib/matplotlib/axes/_base.py

```diff
@@ -1595,9 +1595,19 @@ def axis(self, *v, **kwargs):
 
         Parameters
         ----------
-        v : List[float] or one of the strings listed below.
+        v : List[float] or bool or one of the strings listed below.
             Optional positional-only argument
 
+            If a bool,
+
+            ======== ==========================================================
```

Comment:
A table here seems overkill.

Reply:
okay, I could remove the table!
Though that was just to preserve the formatting similar to that already on the page!

Reply:
I'd say, 

    If a bool, turns axis lines and labels on or off.

Later inside the string table you can also refer to it
```
             Value    Description
             ======== ==========================================================
             'on'     Turn on axis lines and labels. Same as ``True``.
             'off'    Turn off axis lines and labels. Same as ``False``.

```